scripts/create-config.sh

```diff
@@ -22,6 +22,13 @@ driftconfig push -f $CONFIG
 
 driftconfig register
 
+if ! command -v expect; then
```

Comment:
Good stuff, but I would put this higher up so it doesn't run half the steps first and then fails.

Reply:
Moved to the top of the file after #!/usr/bin/env bash thingy